.gitignore

```diff
@@ -44,12 +45,6 @@ deps/dpdk*
 # OSX
 .DS_Store
 
-# Protobuf
```

Comment:
Needs one slight fix to `sample_plugin/modules/sequential_update.h`:
```
-#include "protobuf/supdate_msg.pb.h"
+#include "pb/supdate_msg.pb.h"
```
The generated protobufs go in the `plugin_pb` directory as usual, though, so there's no need to `.gitignore` them.  I'll add the fix as a separate commit, at least for now.

(Note possible drawback: however many plugins you add, all go into `plugin_pb`.  But they also share one big module namespace in BESS itself, so ... 🤷‍♂️ it's no worse than it is for other reasons.)